src/pages/AddProperty/AddProp.jsx

```diff
@@ -123,7 +124,9 @@ class AddProp extends React.Component {
             <div>
                 <TopNav />
                 <div className="saveDraft-publish-buttons">
-                    <button>Save As Draft</button>
+                    <button
+                        onClick={() => this.setState({draft: true})}>
```

Comment:
I think the easiest way is to call the `submitProperty = (bool) => {`... function and pass in either true or false depending on which button was clicked.

Then `draft: bool`

Reply:
Lemme know if you have any questions on that